docs/sdkx-react-native/tracking.mdx

```diff
@@ -0,0 +1,170 @@
+
+Pass this Map to `showConversation` API. For eg:
+
+```javascript
+const apiConfig = {
+    tags: {"foo", "bar"},
+};
+showConversation(apiConfig);
+```
+
+Similarly, you can pass this configuration to other APIs (ie. `showConversation`, `showFAQsWithConfig`, `showFAQSectionWithConfig`).
+
+<Admonition type="info" title="Note">
+
+**On tag names & compatibility**
+
+- Tags must be pre-created in the Helpshift Dashboard (Settings → Tags), otherwise, they will be ignored.
+- The attached tags must exactly match the tags present on the dashboard.
+
+</Admonition>
+
+## Set Custom Metadata {#set-custom-metadata}
+
+If you want to set Custom Metadata at the time of Issue creation, follow the steps.
+
+1. Define your custom metadata `Map` as key-value pairs.
+2. Add the custom metadata `Map` in the map expected in the `Parameters` argument with `customMetadata` as the key.
+
+Pass this Map to the following APIs:
+
+`showConversation` API,
+
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showConversation(apiConfig);
+```
+
+`showFAQsWithConfig` API,
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showFAQsWithConfig(apiConfig);
+```
+
+`showFAQSectionWithConfig` API
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showFAQSectionWithConfig(apiConfig);
+```
+
+## Set Custom Issue Fields {#set-custom-issue-fields}
+
+If you want to set Custom Issue Fields at the time of Issue creation, follow the steps.
+
+1. Initialise a top-level custom issue fields `Map`
+2. Define your custom issue field `Map`
+3. Add the `"type"` and `"value"` for that custom issue field
+   You can find the `type` in this constants class `HelpshiftCustomIssueFieldParameter`
+
+
+Pass this Map to `showConversation` API.
+
+Similarly, you can pass this Map to other APIs (ie. `showConversation`, `showFAQsWithConfig`, `showFAQSectionWithConfig`).
+
+The following are the valid values for the `type` key of a Custom Issue Field.
+
+- NewSingleLineParameter
+- NewMultiLineParameter
+- NewNumberParameter
+- NewCheckboxParameter
+- NewDropdownParameter
+- NewDateParameter
+
+Compatibility table for type and values:
+
+| Type                   | Value    | Comments                                                        |
+| ---------------------- | -------- | --------------------------------------------------------------- |
+| NewSingleLineParameter | string   | Character limit of 255                                          |
+| NewMultiLineParameter  | string   | Character limit of 100,000                                      |
```

Comment:
All the keys are not declared in react native, can we suggest value instead of these keys.

Reply:
Updated.